static/js/components/applications/detail_sections.js

```diff
@@ -259,6 +259,13 @@ export const BootcampStartDetail = (
   props: BootcampStartProps
 ): React$Element<*> => {
   const { ready, fulfilled, applicationDetail } = props
+  let novoedUrl = null
+  if (
+    SETTINGS.novoed_base_url &&
+    applicationDetail.bootcamp_run.novoed_course_stub
+  ) {
+    novoedUrl = `${SETTINGS.novoed_base_url}/#!/courses/${applicationDetail.bootcamp_run.novoed_course_stub}/home`
```

Comment:
1. We should have a helper function that produces this URL so we're not repeating ourselves here, in `ApplicationDashboardPage.js`, and anywhere else we end up needing a NovoEd URL. It's fine to keep the current test case functionality below where it tests for the exact string.

```javascript
const createNovoEdLinkUrl = (baseUrl: string, stub: string): string => (
 //...
)
```

2. If the `novoedUrl` value is null, we should not render a link. 

```javascript
{(ready && fulfilled && novoedUrl) ? (
        <div className="col-12 col-sm-5 text-sm-right">
          <a href={novoedUrl} target="_blank" rel="noopener noreferrer">
            Start Bootcamp
          </a>
        </div>
   // ...
```